ui/app/pages/confirmation/confirmation.js

```diff
@@ -138,16 +133,11 @@ export default function ConfirmationPage() {
   }, [pendingConfirmation, t, dispatch]);
 
   useEffect(() => {
-    const environmentType = getEnvironmentType();
     // If the number of pending confirmations reduces to zero when the user
     // is in the fullscreen or popup UI, return them to the default route.
```

Comment:
Nit: This comment should probably be updated since this now applies to the notification UI as well